docs/wasmer-js/node-modules/examples/transforming-wasi-modules/transforming-wasi-modules.md

```diff
@@ -6,69 +6,201 @@ sidebar_label: Transforming WASI Modules
+
+1. Change into some development directory
+
+    ```bash
+    $ cd <some_development directory>
+    ```
+
+1. Clone this entire repo
+
+    ```bash
+    $ git clone https://github.com/wasmerio/docs.wasmer.io.git
+    ```
+
+1. Change into the `transforming-wasi-modules` directory
+
+    ```bash
+    $ cd docs.wasmer.io/docs/wasmer-js/node-modules/examples/transforming-wasi-modules
+    ```
+
+1. Install the required `npm` dependencies
+
+    ```bash
+    $ npm install
+    ```
+
+1. Start `parcel` 
+
+   ```bash
+   $ parcel index.html
+   ```
+
+1. Point your browser to [`http://localhost:1234`](http://localhost:1234) and you should see `Standard Output: Done!` appear both on the browser screen and in the JavaScript console
+
+# The `clock_time_get` WebAssembly Module
+
+In this case, the native "OS" function we want to call is `clock_time_get`.  To understand how we should call this function, we need look inside the WebAssembly module `clock_time_get.wasm`.  When converted to [WebAssembly Text](https://webassembly.github.io/spec/core/text/index.html) format, the first few lines of this module looks like this:
+
+```WebAssemblyText
+(module
+  (type $t0 (func (param i32 i64 i32) (result i32)))
+  (type $t1 (func (param i32 i32 i32 i32) (result i32)))
+  (type $t2 (func))
+  (import "wasi_unstable" "clock_time_get" (func $wasi_unstable.clock_time_get (type $t0)))
+  (import "wasi_unstable" "fd_write" (func $wasi_unstable.fd_write (type $t1)))
+
+  ;; snip...
+```
+
+On line 2, we can see the declaration of a type definition called `$t0`.  This type definition represents the interface to some `func`tion that takes three, signed integers as parameters and returns an integer.
+
+```WebAssemblyText
+(type $t0 (func (param i32 i64 i32) (result i32)))
+```
+
+Notice the data type of the second parameter; uh oh! Its a 64-bit signed integer!
+
+At the moment, we have no way to directly pass a JavaScript `BigInt` into WebAssembly; therefore, before calling this WebAssembly module, the interface to this function must be transformed
+
+Then on line 5, we can see the declaration of the call to `clock_time_get`:
+
+```WebAssemblyText
+(import "wasi_unstable" "clock_time_get" (func $wasi_unstable.clock_time_get (type $t0)))
+```
+
+Two things are important to notice here:
+
+1. The `import` keyword indicates that function `clock_time_get` lives in an external module called `wasi_unstable`
+
+1. The interface to this function is described by the previously declared type definition `$t0`.  In other words, we know for certain that function `clock_time_get` must be passed an `i64` as its second parameter.
+
+Having established exactly what the interface is to `clock_time_get`, we now know that we cannot call this WASM module without first transforming it.
+
+### Important
```

Comment:
So I went ahead an opened an issue at: https://github.com/wasmerio/docs.wasmer.io/issues/24

That way the `clock_time_get` module makes a bit more sense for an end-user 😄